tests/test-flycheck-popup-tip.el

```diff
@@ -27,7 +27,7 @@
 
 (require 'buttercup)
 (require 'flycheck)
-(require 'undercover-init.el)
```

Comment:
This breaks all tests. Problems is that `undercover-init.el` file provides `undercover-init.el`, not `undercover-init`, so that needed to be changed too.